03-Homework-Css/time-tracking-dashboard-main/style.css

```diff
@@ -0,0 +1,128 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body {
+    font-family: "Rubik";
+}
+p{   
+    font-size: 18px;
+    font-weight: 400;
+    color: hsl(236, 100%, 87%);
+    margin: 18px;
+}
+h1 {       
+    font-size: 40px;
+    font-weight: 400;
+    color: white;
+    margin: 15px;
+}
+h2 {
+    font-size: 30px;
+    font-weight: 500;
+    color: white;
+    margin-left: 15px;   
+}
+a {
+    color: hsl(236, 100%, 87%);
+    text-decoration: none;   
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between; 
+}
+.card {   
+    background-color: hsl(226, 43%, 10%);
+    margin: 20px;
+    display: flex;
+    justify-content: center;    
+}
+.card-report {
+    width: 250px;
+    height: 460px;
+    margin: 50px 10px 50px 40px;
+    background-color: hsl(235, 46%, 20%);
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+}
+.card-header {
+    width: 250px;
+    height: 320px;
+    background-color: hsl(246, 80%, 60%);
+    border-radius: 10px;
+}
+.card-user-photo {
+    width: 110px;
+    height: 110px;
+    margin: 20px;
+    border-radius: 100px;
+    border: 3px solid white;    
+}
+.card-description {
+    width: 250px;
+    height: 100px;   
+}
+.card-results {   
+    max-width: 680px;
+    max-height: 480px;
+    margin-top: 50px;
+    display: flex;     
+    justify-content: space-around;
+    flex-flow: row wrap;
+}
+.card-results-work-header, .card-results-play-header, .card-results-study-header, .card-results-exercise-header, .card-results-social-header, .card-results-self-header {
+    width: 200px;
+    height: 25px;
+    border-radius: 10px 10px 0px 0px;     
+}
+.card-icon1, .card-icon2, .card-icon3, .card-icon4, .card-icon5, .card-icon6 {
+    width: 20%; 
```

Comment:
Daca astea is imaginile de iconite, de aici vine distorsonarea, o imagine/iconita trebuie sa isi pastreze ratio-ul initial (ratio inaltime X latime) daca maresti una dintre ele se distorsoneaza, cum ai dat tu width si height cat ai considerat. Incearca sa o incadrezi acolo chiar daca nu e perfect, e taiata, nu te lega de height..doar de width.